src/azure-cli/azure/cli/command_modules/mysql/_validators.py

```diff
@@ -150,7 +150,7 @@ def _mysql_import_data_source_type_validator(data_source_type, data_source_backu
 
 
 def _mysql_import_mode_validator(mode):
-    allowed_values = ['offline']
```

Comment:
This seems a breaking change. Does customer need to be aware?

Reply:
It is a new feature and in the last PR, it was included in release doc so it should be fine